src/store/index.js

```diff
@@ -1,13 +1,16 @@
 import Vue from 'vue'
 import Vuex from 'vuex'
+import device from './modules/device'
+import blog from './modules/blog'
 
 Vue.use(Vuex)
 
 const store = new Vuex.Store({
     // !this is just for testing purpose
```

Comment:
remove this comment, as this is not relevant now

Reply:
done